abixen-platform-core/src/main/resources/liquibase/changelog/comment-changelog.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd"
+        logicalFilePath="liquibase/changelog/comment-changelog.xml">
+
+    <changeSet author="MariuszKumor" id="comment-v1">
+        <createTable tableName="comment">
+            <column name="id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="message" type="varchar(40)">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
Please change constraintName="fk__comment_parent_id" to constraintName="fk__comment_parent_id__comment__comment" - then will suit to pattern used  in the application